Fix nullability wg link (#89)
graphql/nullability-wg: notes/2025/2025-02.md

# Nullability WG - January 2025

**Watch the replay:**

> [GraphQL Working Group Meetings on YouTube](https://www.youtube.com/watch?v=HSbChLT9Itg&list=PLP1igyLx8foFPThkIGEUVbne2_DBwgQke)
> 
> <!--
> 
> For future notes files, use this link instead:
> 
> **Watch the replay:**
> [GraphQL Working Group Meetings on YouTube](https://www.youtube.com/playlist?list=PLP1igyLx8foFPThkIGEUVbne2_DBwgQke)

Reviewer: ```suggestion
[GraphQL Nullability WG Meetings on YouTube](https://www.youtube.com/watch?v=HSbChLT9Itg&list=PLP1igyLx8foFPThkIGEUVbne2_DBwgQke)

<!--

For future notes files, use this link instead:

**Watch the replay:**
[GraphQL Nullability WG Meetings on YouTube](https://www.youtube.com/playlist?list=PLP1igyLx8foFPThkIGEUVbne2_DBwgQke)
```

> # Client Controlled Nullability WG - January 2025

Reviewer: ```suggestion
# Nullability WG - January 2025
```

**Watch the replay:**

> [GraphQL Working Group Meetings on YouTube](https://www.youtube.com/watch?v=HSbChLT9Itg)

Reviewer: ```suggestion
[GraphQL Working Group Meetings on YouTube](https://www.youtube.com/watch?v=HSbChLT9Itg&list=PLP1igyLx8foFPThkIGEUVbne2_DBwgQke)

<!--

For future notes files, use this link instead:

**Watch the replay:**
[GraphQL Working Group Meetings on YouTube](https://www.youtube.com/playlist?list=PLP1igyLx8foFPThkIGEUVbne2_DBwgQke)

-->
```
